packages/fxa-payments-server/src/routes/Product/index.stories.tsx

```diff
@@ -31,22 +37,126 @@ function init() {
         ]
       }} />
     ))
+
+  storiesOf('routes/Product/page load', module)
+    .add('profile loading', () => (
+      <ProductRoute routeProps={{
+        ...MOCK_PROPS,
+        profile: { loading: true, error: null, result: null }
+      }} />
+    ))
+    .add('profile error', () => (
+      <ProductRoute routeProps={{
+        ...MOCK_PROPS,
+        profile: {
+          loading: false,
+          result: null,
+          error: new APIError({ code: 500, message: 'Internal Server Error' }),
+        }
+      }} />
+    ))
+    .add('customer loading', () => (
```

Comment:
Do we expect these different loading screens to ever look different from each other?

Reply:
I don't think so? But maybe? They could, but they don't have to.